ttnn/cpp/pybind11/pytensor.cpp

```diff
@@ -115,31 +115,30 @@ Tensor convert_float_vector_to_tt_tensor(
                 Layout::TILE,
                 layout);
         }
+        auto result_cpu_spec = TensorSpec(
```

Comment:
cpu_spec?

Reply:
Yes, it is the TensorSpec of the result on cpu